modules/ibi-installer-installation-config.adoc

```diff
@@ -81,4 +87,9 @@ additionalTrustBundle: \|
 |`sshKey`|`string`|Specifies the SSH key to authenticate access to the host.
 |`ignitionConfigOverride`|`string`|Specifies a JSON string containing the user overrides for the Ignition config. The configuration merges with the Ignition config file generated by the installation program. This feature requires Ignition version is 3.2 or later.
 
+[NOTE]
+====
+The default partition label is `varlibcontainers`. If you set a different partition label, you must ensure it matches the partition label set in the seed image. For more information, see "Configuring a shared container partition between ostree stateroots".
```

Comment:
I would also advise to make sure that the label they're going to be using matches the one on the already running system, maybe with an example command that shows how to check it?

Reply:
I believe it should it be a little bit different phrase, like
Please make sure that your label you set (default ...) matches the one from machine config in seed

Reply:
I've had another go rephrasing this... please see if it's any better or feel free to suggest something else.

Another (more time-consuming) option is to create a Troubleshooting section and highlight this issue. It would probably need some sample error output too that would indicate a partition mount failure because of label mismatches.